tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/SharePointInfo/Search.cs

```diff
@@ -2025,6 +2025,11 @@ private KeywordQuery AssignKeywordQueryValues(KeywordQuery keywordQuery, List<st
                     keywordQuery.EnableSorting = true;
                     keywordQuery = GetSortByProperty(keywordQuery, searchObject, isMatterSearch);
                 }
+                if(keywordQuery.QueryText.Length > 4000)

```

Comment:
If the length of the query text is larger that 4096 characters, SPO search is throwing the error. So added this condition